docs/appendices/issues.rst

```diff
@@ -0,0 +1,485 @@
+.. _issues:
+
+======
+Issues
```

Comment:
Can we find a different or more concrete name here as these are not general issues but resiliency related issues?

Reply:
good catch! I've re-worked things. thanks!